src/main/java/com/github/manolo8/darkbot/core/entities/Box.java

```diff
@@ -135,4 +141,61 @@ public Beacon(int id, long address) {
             super(id, address);
         }
     }
+
+    public static class FakeBox extends Box implements eu.darkbot.api.game.entities.FakeEntity.FakeBox {
+        private static int CURR_ID = Integer.MIN_VALUE;
```

Comment:
needs to be shared by all fake entities, to avoid duplicate ids. Just have this sit in some util where you can `EntityUtil.allocateFakeId()` and it returns CURR_ID++ or similar